node/src/components/consensus/protocols/highway.rs

```diff
@@ -357,22 +357,19 @@ impl<I: NodeIdT, C: Context + 'static> HighwayProtocol<I, C> {
     /// Returns the median round exponent of all the validators that haven't been observed to be
     /// malicious, as seen by the current panorama.
     /// Returns `None` if there are no correct validators in the panorama.
-    pub(crate) fn median_round_exp(&self) -> Option<u8> {
+    fn median_round_exp(&self) -> Option<u8> {
```

Comment:
Even by just reporting _our_ round length, we are giving away the information that this node is a validator. And in addition, and attacker could compare the reported round lengths with the round exponents specified in the different validators' units, to eventually find out _which_ validator that node is.

Instead, we could always report the median round length? Then every node in the network would return the same information.

Alternatively, we could return a whole list of the different validators' most recent round lengths, or a histogram. That would also be returned by every node in the network.